specification/search/resource-manager/Microsoft.Search/preview/2019-10-01-preview/search.json

```diff
@@ -1106,6 +1417,190 @@
+          "description": "The name of the private link resource."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The resource type."
+        },
+        "properties": {
+          "$ref": "#/definitions/PrivateLinkResourceProperties",
+          "readOnly": true,
+          "description": "Describes the properties of an available private link resource for the Azure Cognitive Search service."
+        }
+      },
+      "description": "Describes an available private link resource for the Azure Cognitive Search service."
+    },
+    "PrivateLinkResourceProperties": {
+      "properties": {
+        "groupId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The group ID of the private link resource."
+        },
+        "requiredMembers": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The list of required members of the private link resource."
+        },
+        "requiredZoneNames": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The list of required DNS zone names of the private link resource."
+        },
+        "shareablePrivateLinkResourceTypes": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ShareablePrivateLinkResourceType"
+          },
+          "description": "The list of available shareable private link resource types."
+        }
+      },
+      "description": "Describes the properties of an available private link resource for the Azure Cognitive Search service."
+    },
+    "ShareablePrivateLinkResourceType": {
+      "properties": {
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The name of the shareable private link resource type."
+        },
+        "properties": {
+          "$ref": "#/definitions/ShareablePrivateLinkResourceProperties",
+          "readOnly": true,
+          "description": "Describes the properties of an available shareable private link resource that can be managed by the Azure Cognitive Search service."
+        }
+      },
+      "description": "Describes an available shareable private link resource that can be managed by the Azure Cognitive Search service."
+    },
+    "ShareablePrivateLinkResourceProperties": {
+      "properties": {
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The resource type from the provider of resource the shareable private link resource is available for."
+        },
+        "groupId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The group id from the provider of resource the shareable private link resource is available for."
```

Comment:
The resource provider group id for the *resource* that has been on boarded to private link service, supported by Azure Cognitive Search.